### ISSUE TITLE
ivy-20161213.719 font face is wrong. The candidates are invisible.

### ISSUE THREAD

**Issue by redguardtoo** [abo-abo/swiper#828]
Here is the screenshot after `M-x counsel-M-x`:
![emacs-nq8](https://cloud.githubusercontent.com/assets/184553/21181634/f0cd7e8e-c252-11e6-8626-28331111ec9b.png)

- emacs v24.4.1
- can be reproduced on GUI and console version
- can be reproduced after I comment out the color-theme setup

**Comment by joedicastro** [comment]
I think this is related with #654 and there is already a pull request to fix it

**Comment by redguardtoo** [comment]
This version is fine,
https://github.com/redguardtoo/myelpa/blob/master/ivy-20161204.48.tar

**Comment by abo-abo** [comment]
Please check again with the latest master.

**Comment by redguardtoo** [comment]
It's not fixed in ivy-20161215.543. I think the major issue exists in ivy, because my own command which only use `ivy-read` has same issue.

**Comment by redguardtoo** [comment]
`(setq ivy-display-style 'fancy)` will fix the issue, I'm using Emacs 24.4.1, as mentioned before.

The bad commit is ca84f24521fac922d41ccfa8b42632b10acc9dd3, I tested both ca84f24521fac922d41ccfa8b42632b10acc9dd3 and HEAD version.

The root cause is below code,
```lisp
(defun ivy--format-minibuffer-line (str)
  (when (eq ivy-display-style 'fancy)
    (funcall ivy--highlight-function (copy-sequence str))))
```

I don't understand, if I'm not using fancy style, the function refuse to render minibuffer?

**Comment by abo-abo** [comment]
Thanks, please test.
